src/pages/components/select.mdx

```diff
@@ -46,7 +46,7 @@ import SelectDefaultCode from '!!raw-loader!../../examples/dropdowns/SelectDefau
   <SelectDefault />
 </CodeExample>
 
-### Hint
+### Hint copy
```

Comment:
@rossmoody this was the copy update we discussed in order to avoid a same-level nav collision with the `API > Hint` below. Depending on what you figure out with @vsri1137, we can restore this and turn off the `markdownlint` rule (as our heading parser handles duplicates) or align with copy standards.

Reply:
This seems like a great temporary band-aid to me.